client/landing/stepper/declarative-flow/internals/steps-repository/goals/index.tsx

```diff
@@ -8,6 +9,18 @@ import SelectGoals from './select-goals';
 import type { Step } from '../../types';
 import './style.scss';
 
+type TracksGoalsSelectEventProperties = {
+	goals: string;
+	write?: number;
+	promote?: number;
+	sell?: number;
+	difm?: number;
+	import?: number;
+	other?: number;
+	ref?: string;
+	intent: string;
```

Comment:
We could use `Onboard.SiteIntent` type here.

Reply:
Keeping `string` for now because of uncertainty around `import` with Plan B, and the `SiteIntent` enum doesn't have `import`.